test/config/override-configs/postfix-main.cf

```diff
@@ -2,3 +2,6 @@ max_idle = 600s
 # this is a comment
     # this is also a comment
 readme_directory = /tmp
+
+# This parameter is referenced by an override in `postfix-master.cf`:
+custom_parameter = cidr:{{!172.16.0.42 REJECT}}, permit_sasl_authenticated, reject
```

Comment:
The test adds this `postfix-main.cf` custom parameter, which `postfix-master.cf` uses for `smtpd_client_restrictions` value. This is the same as what the referenced discussion was using, it just prepends that `cidr` inline table to reject any IP that is not `172.16.0.42`.

With the `postfix.sh` change to swap the order we apply `postfix-main.cf` and `postfix-master.cf`, the test demonstrates that we can correctly resolve/expand the `$custom_parameter` value in the updated `/etc/postfix/master.cf` file 👍